data/beaconrestapi/src/test/java/tech/pegasys/teku/beaconrestapi/handlers/v1/events/EventTest.java

```diff
@@ -38,10 +38,16 @@ public static Stream<Arguments> getEventTypes() {
 
     final Spec spec = TestSpecFactory.createMinimalPhase0();
     final DataStructureUtil dataStructureUtil = new DataStructureUtil(spec);
+    final Spec specElectra = TestSpecFactory.createMinimalElectra();
+    final DataStructureUtil dataStructureUtilElectra = new DataStructureUtil(specElectra);
     return Stream.of(
         Arguments.of(
-            "AttestationEvent",
+            "AttestationEvent Phase0",
```

Comment:
oh i wouldn't expect these names to be what we want, i guess ultimately this needs to follow the API, but my expectation is we have 1 event...

Reply:
i did try attestation stream and this seems to not matter at all, so im not sure where this comes in exactly...

Reply:
My understanding was that it's just an internal name. We can doublecheck (we could also remove the additional test tbh)

Reply:
yeh it looks ok (didnt re-run test, but i think we're ok)

Reply:
since we don't run an eventTest for each possible fork for a given eventType, I could drop it. And we could improve `EventTest` on a dedicated PR if we want. WDYT?